lib/code_corps_web/controllers/project_user_controller.ex

```diff
@@ -1,35 +1,62 @@
 defmodule CodeCorpsWeb.ProjectUserController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
-  import CodeCorps.Helpers.Query, only: [id_filter: 2]
-
-  alias CodeCorps.{Emails, Mailer, ProjectUser}
+  alias CodeCorps.{Emails, Helpers.Query, Mailer, ProjectUser, User}
 
+  action_fallback CodeCorpsWeb.FallbackController
+  plug CodeCorpsWeb.Plug.DataToAttributes  
+  plug :schedule_email when action in [:update]
+  
   @preloads [:project, :user]
 
-  plug :load_resource, model: ProjectUser, only: [:show], preload: @preloads
-  plug :load_and_authorize_resource, model: ProjectUser, only: [:delete]
-  plug :load_and_authorize_changeset, model: ProjectUser, only: [:create, :update], preload: @preloads
-  plug :schedule_email when action in [:update]
-  plug JaResource
+  # plug :load_and_authorize_resource, model: ProjectUser, only: [:delete]
+  # plug :load_and_authorize_changeset, model: ProjectUser, only: [:create, :update], preload: @preloads
+  
 
-  @spec model :: module
-  def model, do: CodeCorps.ProjectUser
+  @spec index(Conn.t, map) :: Conn.t
+  def index(%Conn{} = conn, %{} = params) do
+    with project_users <- ProjectUser |> Query.id_filter(params) |> Repo.all do
+      conn |> render("index.json-api", data: project_users)
+    end
+  end
 
-  @spec filter(Plug.Conn.t, Ecto.Query.t, String.t, String.t) :: Ecto.Query.t
-  def filter(_conn, query, "id", id_list) do
-    query |> id_filter(id_list)
+  @spec show(Conn.t, map) :: Conn.t
+  def show(%Conn{} = conn, %{"id" => id}) do
+    with %ProjectUser{} = project_user <- ProjectUser |> Repo.get(id) |> Repo.preload(@preloads) do
+      conn |> render("show.json-api", data: project_user)
+    end
   end
 
-  @spec handle_create(Plug.Conn.t, map) :: Ecto.Changeset.t
-  def handle_create(_conn, attributes) do
-    %ProjectUser{} |> ProjectUser.create_changeset(attributes)
+  @spec create(Plug.Conn.t, map) :: Conn.t
+  def create(%Conn{} = conn, %{} = params) do
+    with %User{} = current_user <- conn |> Guardian.Plug.current_resource,
+         {:ok, :authorized} <- current_user |> Policy.authorize(:create, %ProjectUser{}, params),
+         {:ok, %ProjectUser{} = project_user} <- %ProjectUser{} |> ProjectUser.create_changeset(params) |> Repo.insert 
```

Comment:
The old code had `load_and_authorize_changeset` which included preload. I don't quite understand where the preloading should be happening here.
`  plug :load_and_authorize_changeset, model: ProjectUser, only: [:create, :update], preload: @preloads`